cypress/integration/myPatches/patchCard/dropdown_menu.ts

```diff
@@ -105,7 +106,7 @@ describe("Dropdown Menu of Patch Actions", { testIsolation: false }, () => {
 
   it("Toggle patch visibility", () => {
     // "Include hidden" checkbox is not checked and patch is visible
-    cy.getCookie(INCLUDE_HIDDEN_PATCHES).should("not.exist");
+    cy.getCookie(INCLUDE_HIDDEN_PATCHES).should("equal", null);
```

Comment:
I think it might be okay to delete, since Cypress should automatically clear all cookies under test isolation
```suggestion
```